packages/fast-components-styles-msft/src/patterns/input-field.ts

```diff
@@ -1,9 +1,8 @@
-import { horizontalSpacing } from "../utilities/density";
 import { CSSRules } from "@microsoft/fast-jss-manager";
+import { applyCornerRadius } from "../utilities/border";
 import { DesignSystem } from "../design-system";
-import { format, toPx } from "@microsoft/fast-jss-utilities";
+import { format, subtract, toPx } from "@microsoft/fast-jss-utilities";
```

Comment:
we do not appear to be using subtract, please remove

Reply:
removed